freqtrade/commands/cli_options.py

```diff
@@ -82,6 +82,11 @@ def __init__(self, *args, **kwargs):
         help='Reset sample files to their original state.',
         action='store_true',
     ),
+    "recursive": Arg(
```

Comment:
you'll have to add this to `configuration.py` "someplace" (just look for a spot near the other strategy options).

options will not automatically transfer to the configuration - it's a one-line addition though.